netbox/project-static/styles/overrides/_tabler.scss

```diff
@@ -70,6 +76,47 @@ table th.orderable a {
   color: $rich-black!important;
 }
 
+// Add button focus state
+.btn:focus {
+  border: 1px solid var(--tblr-primary-fg);
+  outline: 2px solid var(--tblr-primary)!important;
+}
+
+// Fix secondary combo button contrast
+.btn-outline-secondary {
+  &.active {
+    color: var(--tblr-primary-fg);
+  }
+  &:hover {
+    color: var(--tblr-primary-fg);
+  }
+}
```

Comment:
This doesn't seem to have any effect? A button with `btn-outline-secondary` appears to render the same whether or not this rule is in place AFAICT.